docs/src/layouts/links.js

```diff
@@ -148,7 +148,7 @@ const ComponentsLinks = [
       {
         id: 'FORM_VALIDATION',
         route: routes.FORM_VALIDATION,
-        children: 'Validation',
+        children: 'Form validation',
```

Comment:
This just renames the page in the side-bar. I thought it makes more sense for it to live with Form elements instead of being right at the bottom of the list